README.md

```diff
@@ -1,42 +1,138 @@
-# Scotch
+# EasyVCR
 
-### Disclaimer
-This is a fork of the original [Scotch by Martin Leech](https://github.com/mleech/scotch), updated to address outstanding functionality issues, as well as introduce a handful of new features. This version of Scotch is a C# rewrite of the original F# version.
+#### Disclaimer
+This is a distant fork of the original [Scotch by Martin Leech](https://github.com/mleech/scotch), completely rewritten in C# and enhanced with several new features and performance improvements.
 
-## What is Scotch?
-Scotch is a library for recording and replaying HTTP interactions in your test suite. 
+## What is EasyVCR?
```

Comment:
Let's remove this title and have the next sentence/paragraph be the first thing after the header in the README